rules/lfi-os-files.data

```diff
@@ -161,6 +319,8 @@
 .sql.xz
 .sql.yz1
 .sql.z
+# CVE-2023-49103
+phpinfo.php
```

Comment:
Technically, the CVE is about the `GetPhpInfo.php` filename. But in the end, this is about information disclosure and probably makes sense to block, as we block other stuff that is more or less of the same type.